src/tools/profile/read_profile.c

```diff
@@ -268,6 +268,16 @@ static void show_profile_data_log(profile_data_t *data, options_t *opts)
 
     kh_init_inplace(request_ids, &reqids);
 
+#ifdef __clang_analyzer__
+    /* clang analyzer doesn't have a better way to suppress
```

Comment:
can we just exclude all failing code without extra settings?

Reply:
maybe we can exclude whole files? like khash.h?

Reply:
yes, that would be good to do for all 3-d party codes.

Reply:
but other places will be affected then (we couldn't catch errors related to khash usage in our code)

Reply:
@dmitrygladkov can we add some suppressions inside khash.h code?

Reply:
@yosefe I'm looking for the way how to fix/suppress this

Btw, let me explain what clang thinks about khash code
Clang static analyzer has the following sequence:
```
<inside khash.h>

int j = 1;
...
if (h->size >= <some thresh>) { /* <= clang evaluated to false */
    j = 0;
} else {
    /* takes this branch */
    new_flags = ucs_malloc(...);
}

if (j) { /* <= clang evaluated to false. i.e. doesn't take this branch */
    ...
    ucs_free(h->flags);
    h->flags = new_flags;
}
```
After that it reports that we dereference `h->flags` while this is `NULL`

Even if I do:
```
<inside khash.h>

int j = 1;
...
if (h->size >= <some thresh>) { /* <= clang evaluated to false */
    j = 0;
} else {
    /* takes this branch */
    new_flags = ucs_malloc(...);
}

if (h->size < <some thresh>) { /* <= clang evaluated to false. i.e. doesn't take this branch */
    ...
    ucs_free(h->flags);
    h->flags = new_flags;
}
```